website/blog/2024-05-17-whats-next.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: whats-next
+title: What's next for MACI?
```

Comment:
Just a suggestion, but I'd like to update this to something more descriptive - maybe something like "Upcoming grants for MACI protocol improvements"? Something that would get people excited about the idea of applying for grants to work on MACI

Reply:
Yeah maybe "Someone, help... please?"